mysql/docker-entrypoint-initdb.d/02_data.sql

```diff
@@ -15,7 +15,9 @@ INSERT INTO
         ironman,
         hardcore_ironman,
         ultimate_ironman,
-        normalized_name
+        normalized_name,
+        feedback_received,
+        feedback_given

```

Comment:
this does not exist in Players table

Reply:
uhhhhh so i did this - 

https://github.com/RusticPotatoes/public-api/blob/909b9bca5580c5d74cf9607a3e9571833f8f9e2a/mysql/docker-entrypoint-initdb.d/01_tables.sql#L17-L19

https://github.com/RusticPotatoes/public-api/blob/909b9bca5580c5d74cf9607a3e9571833f8f9e2a/mysql/docker-entrypoint-initdb.d/01_tables.sql#L114C1-L119

Reply:
resolved by

[remove foreign keys not needed for given and received](https://github.com/Bot-detector/public-api/pull/25/commits/ab9a14783294c2231895190c10eaa89a7d659b1a)
[ab9a147](https://github.com/Bot-detector/public-api/pull/25/commits/ab9a14783294c2231895190c10eaa89a7d659b1a)

and 

[remove PredictionFeedbackOut and foreign keys](https://github.com/Bot-detector/public-api/pull/25/commits/fa684a8c63e7611eacd3c8e1b5b4c4eb3f9dc405)

as tests are still passing after removal